### PR TITLE
Looser Validation

### PR DESCRIPTION
**Pull request by graphemecluster**
Now checks pass!

### CODE REVIEW

File: word.csv [hunk 1]
```diff
@@ -6077,7 +6076,6 @@ char,jyutping
 依便,ji1 bin6
 依傍,ji1 bong6
 依勻,ji1 wan4
-依哇鬼叫,wi1 waa1 gwai2 giu3
```
Comment (graphemecluster) [inline word.csv:6080]: 已有「喴嘩鬼叫」、「喂哇鬼叫」

File: word.csv [hunk 2]
```diff
@@ -20432,7 +20430,6 @@ char,jyutping
 塊規,faai3 kwai1
 塊閪,faai3 hai1
 塊體,faai3 tai2
-塌Q,taap3 kiu1
```
Comment (graphemecluster) [inline word.csv:20435]: Should be moved to `lettered.dict.yaml`

File: fixed_expressions.csv [hunk 3]
```diff
@@ -115,11 +115,9 @@ char,jyutping
 不費吹灰之力,bat1 fai3 ceoi1 fui1 zi1 lik6
 不食人間煙火,bat1 sik6 jan4 gaan1 jin1 fo2
 且聽下回分解,ce2 ting1 haa6 wui4 fan1 gaai2
-世上無難事，只怕有心人,sai3 soeng5 mou4 naan4 si6 zi2 paa3 jau5 sam1 jan4
 亞茂整餅，冇嗰樣整嗰樣,aa3 mau6 zing2 beng2 mou5 go2 joeng2 zing2 go2 joeng2
 人一世，物一世,jan4 jat1 sai3 mat6 jat1 sai3
 人不可貌相，海水不可斗量,jan4 bat1 ho2 maau6 soeng3 hoi2 seoi2 bat1 ho2 dau2 loeng6
-人不爲己，天誅地滅,jan4 bat1 wai6 gei2 tin1 zyu1 dei6 mit6
```
Comment (hfhchan) [inline fixed_expressions.csv:122]: Why is this deleted btw

Comment (hfhchan) [reply fixed_expressions.csv:122]: Should be delete the one in words.csv and keep the one in fixed_expressions

**Review by hfhchan — CHANGES_REQUESTED** [review]
Hmm I think we should manually clean up the duplicates a bit before we fix the de-duper

File: fixed_expressions.csv (hunk 3 above)
Comment (hfhchan) [reply fixed_expressions.csv:122]: Do you mind making the de-duper just output a warning for now, and then killing the last de-dupe commit? Then we can manually fix them incrementally. Otherwise, this would undo quite a bit of cleanup I just had these two weeks :/

Comment (graphemecluster) [reply fixed_expressions.csv:122]: OK

File: scripts/dedup_across_files.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+seen = set()
+seen_add = seen.add
+
+for filename in ['word.csv', 'trending.csv', 'fixed_expressions.csv', 'phrase_fragment.csv']:
```
Comment (graphemecluster) [inline scripts/dedup_across_files.py:4]: @hfhchan The priority list is here, feel free to change the order if you are not comfortable with it

Comment (hfhchan) [reply scripts/dedup_across_files.py:4]: `'fixed_expressions.csv', 'phrase_fragment.csv', 'trending.csv', 'word.csv'` thanks

**Comment by hfhchan** [comment]
i think i resolved them all. do you mind to rebase and re-run the dedupe, see if anything comes up?

**Comment by graphemecluster** [comment]
@hfhchan Done

**Review by hfhchan — APPROVED** [review]
